ui/apps/platform/src/Containers/Policies/Wizard/Step3/policyCriteriaDescriptors.tsx

```diff
@@ -249,7 +249,7 @@ export type Descriptor =
 
 export type TableModalDescriptor = {
     type: 'tableModal';
-    component: React.ReactNode;
+    component: typeof ImageSigningTableModal;
```

Comment:
There is only one place the `TableModalDescriptor` type is used or instantiated, and that is with a value of `ImageSigningTableModal` passed to the component field. Instead of building up a general type here, I used the exact type of that component to limit the scope of changes.